### PR TITLE
feat(Root): normalize border variables

### PR DESCRIPTION
**Pull request by xaviermariaconejo**
Normalize border variables

### CODE REVIEW

**Review by adriadoz — APPROVED** [review]

**Comment by turolopezsanabria** [comment]
I suggest we never change the values where $bdrs-base is until we control the other values.
In other words, $bdrs-base is valid value everywhere it is.

All the other changes are ok (/)

File: src/components/_select.scss [hunk 1]
```diff
@@ -15,7 +15,7 @@
   @include appearance(none);
   background-color: $bgc-select;
   border: 0;
-  border-radius: $bdrs-base;
+  border-radius: $bdrs-xl;
```
Comment (turolopezsanabria) [inline src/components/_select.scss:18]: I suggest not changing this value. Please leave $bdrs-base

File: src/components/_select.scss [hunk 2]
```diff
@@ -37,8 +37,8 @@
 
   &-icon-wrap {
     background-color: $bgc-select-icon-wrap;
-    border-bottom-right-radius: $bdrs-base;
-    border-top-right-radius: $bdrs-base;
+    border-bottom-right-radius: $bdrs-xl;
+    border-top-right-radius: $bdrs-xl;
```
Comment (turolopezsanabria) [inline src/components/_select.scss:41]: I suggest not changing this value. Please leave $bdrs-base

File: src/settings/_box-style.scss [hunk 3]
```diff
@@ -1,15 +1,12 @@
 // --- Box Style --- //
 
 // Border radius
-$bdrs-base: 8px !default;
```
Comment (turolopezsanabria) [inline src/settings/_box-style.scss:4]: we use bdrs-base in many places, please don't remove

**Pull request merged**